Extensions/activity_plus.css

```diff
@@ -1,13 +1,13 @@
-.xkit-activity-plus-condensed-item.xkit-activity-plus-yellow-mode-on {
+.ui_note.xkit-activity-plus-condensed-item.xkit-activity-plus-yellow-mode-on {
```

Comment:
would prefer to use !important  rather then having a specificity battle, because we're intentionally overriding things